### PR TITLE
add libhugetlbfs test readme

### PR DESCRIPTION
**Pull request by ftjahjad**
@spbnick 

Please review libhugetlbfs test readme PR.

### CODE REVIEW

File: vm/hugepage/libhugetlbfs/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# libhugetlbfs test suite
+libhugetlbfs test provides huge pages of memory unit testing for the libhugetlbfs file system. The source code for libhugetlbfs wrapper can be found at https://github.com/libhugetlbfs/libhugetlbfs.git \
```
Comment (veruu) [inline vm/hugepage/libhugetlbfs/README.md]: @ftjahjad can you please remove the `\` at the end of the line here? I don't see any use for it

Comment (ftjahjad) [reply vm/hugepage/libhugetlbfs/README.md]: @Veruu 
`\`  at the end  to align "test maintainer......." to next line on my understanding,
If  without `\`, "test maintainer...." will be display next after it, not on next line :-D

Comment (veruu) [reply vm/hugepage/libhugetlbfs/README.md]: Ah, ok! I missed there is no empty line so I thought it's joining the lines together. My bad.

Comment (ftjahjad) [reply vm/hugepage/libhugetlbfs/README.md]: Thanks  @revuu :+1:

**Pull request merged**